### PR TITLE
Update JQuery version to 3.6.0

### PR DESCRIPTION
**Pull request by Yasara-Y**
Fixes https://github.com/wso2/product-is/issues/13432

### CODE REVIEW

File: component/authentication-endpoint/src/main/webapp/libs/jquery_3.6.0/jquery-3.6.0.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 /*!
```
Comment (brionmario) [inline component/authentication-endpoint/src/main/webapp/libs/jquery_3.6.0/jquery-3.6.0.js:1]: Check if we can use the minified version.

File: component/authentication-endpoint/src/main/webapp/emailotpError.jsp [hunk 2]
```diff
@@ -118,7 +118,7 @@
             </p>
         </div>
     </footer>
-    <script src="libs/jquery_3.4.1/jquery-3.4.1.js"></script>
+    <script src="libs/jquery_3.6.0/jquery-3.6.0.js"></script>
     <script src="libs/bootstrap_3.4.1/js/bootstrap.min.js"></script>
     </body>
     </html>
```
Comment (brionmario) [inline component/authentication-endpoint/src/main/webapp/emailotpError.jsp]:
```suggestion
    </html>

```

**Review by brionmario — APPROVED** [review]

File: component/authentication-endpoint/src/main/webapp/emailotpError.jsp (hunk 2 above)
Comment (Yasara-Y) [reply component/authentication-endpoint/src/main/webapp/emailotpError.jsp]: Fixed with  18930158d22e7d1d3f35235063005a473c852aaa

File: component/authentication-endpoint/src/main/webapp/libs/jquery_3.6.0/jquery-3.6.0.js (hunk 1 above)
Comment (Yasara-Y) [reply component/authentication-endpoint/src/main/webapp/libs/jquery_3.6.0/jquery-3.6.0.js:1]: After the offline discussions with the team, it was decided to keep it as it is, given that this is a back-port fix and the same set of non-minified files are available in IS5.11.0 and public fix as well. Furthermore IS 5.7.0 will be having its end-of-lifecycle in August 2022. Hence the effort to change all JQuery js files to the corresponding minified versions across all 9 repositories involved will be not worth the time and effort.

**Pull request merged**